internal/consensus/replay_test.go

```diff
@@ -579,10 +579,10 @@ func setupSimulator(ctx context.Context, t *testing.T) *simulatorTestSuite {
 	ensureNewRound(t, newRoundCh, height+1, 0)
 
 	sim.Chain = make([]*types.Block, 0)
-	sim.Commits = make([]*types.Commit, 0)
+	sim.ExtCommits = make([]*types.ExtendedCommit, 0)
```

Comment:
I see that this pattern is pre-existing, but I'm not sure why we are allocating anything for these. If we want empties, the default is fine. These aren't protobuf types so there's no obvious marshaling issue. Could we drop these assignments?  (The allocation isn't required for append)

Reply:
It looks like the code here intended to reset these values on purpose, as `sim` is used extensively prior to this in ways that would populate these fields.

Are you suggesting rather preferring a reset like the following?

```golang
sim.ExtCommits = []*types.ExtendedCommit{}
```